crates/binstalk/Cargo.toml

```diff
@@ -18,6 +18,7 @@ command-group = { version = "2.1.0", features = ["with-tokio"] }
 compact_str = { version = "0.7.0", features = ["serde"] }
 detect-targets = { version = "0.1.7", path = "../detect-targets" }
 either = "1.8.1"
+gix = { version = "0.46.0", features = ["blocking-http-transport-reqwest-rust-tls"], optional = true }
```

Comment:
(unrelated) if we're using all of gix here we can probably use bosion at low cost...

Reply:
Yeah I was also thinking about this, but then we need to make sure `gix` used in `build-dependencies` are unified with `dependencies`, otherwise we would still end up with two different builds of `gix`.